src/Compiler/CompilerOptions.cpp

```diff
@@ -501,14 +501,18 @@ static llvm::cl::opt<std::string, true> parallelizeOpsOpt("parallelize-ops",
 
 static llvm::cl::opt<std::string, true> instrumentSignatureOpt(
     "instrument-signature",
-    llvm::cl::desc("Specify which high-level operations should print their"
-                   " input type(s) and shape(s)\n"
-                   "\"ALL\" or \"\" for all available operations.\n"
-                   "\"NONE\" for no instrument (default).\n"
-                   "\"ops1,ops2, ...\" for the multiple ops.\n"
-                   "e.g. \"onnx.MatMul,onnx.Add\" for MatMul and Add ops.\n"
-                   "Asterisk is also available.\n"
-                   "e.g. \"onnx.*\" for all onnx operations.\n"),
+    llvm::cl::desc(
+        "Specify which high-level operations should print their"
+        " input type(s) and shape(s)\n"
+        "\"ALL\" or \"\" for all available operations.\n"
+        "\"NONE\" for no instrument (default).\n"
+        "\"ops1,ops2, ...\" for the multiple ops.\n"
+        "e.g. \"onnx.MatMul,onnx.Add\" for MatMul and Add ops.\n"
+        "Asterisk is also available.\n"
+        "e.g. \"onnx.*\" for all onnx operations.\n"
+        "If this option is started with \"onnx_node_name\"\n"
+        "the attribute of \"onnx_node_name\", instead of the op name\n"
```

Comment:
missing the `:` in the pattern, as you are looking for this
```
std::string header = "onnx_node_name:";
```

Also you state that "the data values" will be printed. Is that the input data values, the output data values? Please specify.

Reply:
Fixed.